backend/code_review_backend/issues/tests/test_api.py

```diff
@@ -386,15 +395,18 @@ def test_create_issue_bulk_alread_exists(self):
         issues = list(Issue.objects.order_by("created"))
         self.assertEqual(len(issues), 2)
 
-        with self.assertNumQueries(2):
+        with self.assertNumQueries(6):
             response = self.client.post(
                 f"/v1/revision/{self.revision.id}/issues/", payload_2, format="json"
             )
-            self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-            self.assertDictEqual(
-                response.json(),
-                {"issues": [{"hash": ["issue with this hash already exists."]}]},
-            )
+            self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertCountEqual(
+            [issue["hash"] for issue in response.json()["issues"]],
+            ["somemd5hash", "athirdmd5hash"],
+        )
+        new_issues = list(Issue.objects.order_by("created"))
+        self.assertEqual(len(new_issues), 3)
+        self.assertListEqual([i.id for i in issues], [i.id for i in new_issues[:2]])
```

Comment:
It would be cleaner to check the hash value, hardcoding the hashes there: we make 100% sure we only have expected non-duplicated values